package.json

```diff
@@ -3,14 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
```

Comment:
Although there is nothing wrong with using UI libraries, I think it is encouraged too put the topics from the workshop into practice.